### ISSUE TITLE
Can't interpret statement on Macro

### ISSUE THREAD

**Issue by dedmen** [LordGolias/sqf#43]
https://github.com/acemod/ACE3/blob/c9ad92e92d26f929e3cb78a53743585e5ffa20d8/addons/medical_gui/functions/fnc_collectActions3D.sqf#L20-L24
The interesting thing here is that the first 5 work fine. And although they all do the same. The last one fails.
![atom_2018-07-30_10-43-00](https://user-images.githubusercontent.com/3768165/43386927-57f84f8e-93e5-11e8-8a8c-055ec3761666.png)

**Comment by zalexki** [comment]
Yup kinda same here, but got `IndexError: list index out of range`.
Line is : `[_args, {CALLM(gStimulusManager, "postMethodAsync", _this);}] remoteExecCall["call", 0, false];`

**Comment by Vincentimetr** [comment]
If someone is able to fix that, I take it!
